lib/ansible/modules/cloud/ovirt/ovirt_cluster.py

```diff
@@ -65,8 +65,9 @@
     gluster:
         description:
             - "If I(True), hosts in this cluster will be used as Gluster Storage
-               server nodes, and not for running virtual machines."
-            - "By default the cluster is created for virtual machine hosts."
+               server nodes."
+            - "By default the cluster is created for virtual machine hosts only."
```

Comment:
This is not entirely true, if you want to run VMs on a host you need to have `virt` enabled. But `gluster` is related to storage: if `gluster` is disabled, then you can use only classic storage like NFS, FC, iSCSI, but if `gluster` is enabled then you can also use Gluster storage.